templates/zerver/integrations.html

```diff
@@ -1639,7 +1639,8 @@
           <p>To configure Zulip notifications for a particular Redmine project,
             visit the project's Settings page.</p>
           <p>
-            In either case, fill out the bot email address and API key, and the
+            In either case, fill out zulip server by <code>{{ external_api_path_subdomain }}</code>,
```

Comment:
should this be using `external_api_uri_subdomain`?  It looks like the plugin doesn't quite support the trailing `/api`; might make sense to patch the plugin to remove a trailing `/api` if present and then use `external_api_uri_subdomain`.